code/modules/reagents/chemistry/machinery/chem_mass_spec.dm

```diff
@@ -32,36 +33,66 @@ This will not clean any inverted reagents. Inverted reagents will still be corre
 	var/obj/item/reagent_containers/beaker1
 	///Output reagents container
 	var/obj/item/reagent_containers/beaker2
+	///multiplies the final time needed to proccess the chems depending on the laser stock part
+	var/CMS_coefficient = 1
 
 /obj/machinery/chem_mass_spec/Initialize()
 	. = ..()
-	beaker2 = new /obj/item/reagent_containers/glass/beaker/large(src)
```

Comment:
This is an unwelcome behavioral change. Use the `mapload` variable on Initialize, and only add it then.

Reply:
done